apps/prairielearn/src/migrations/20250117000347_variants__variant_seed_nonnull.sql

```diff
@@ -9,6 +9,7 @@ DROP CONSTRAINT IF EXISTS variants_variant_seed_not_null;
 ALTER TABLE variants
 ADD CONSTRAINT variants_variant_seed_not_null CHECK (variant_seed IS NOT NULL) NOT VALID;
 
+-- squawk-ignore constraint-missing-not-valid
```

Comment:
Nit, so that we don't try to blindly copy/paste this into a new migration:

```suggestion
-- When this migration was written, we didn't know to add the constraint in a separate transaction.
-- squawk-ignore constraint-missing-not-valid
```